contracts/base/LendingPool.sol

```diff
@@ -215,13 +228,22 @@ contract LendingPool is Base, LendingPoolInterface {
 
         // Withdraw the tokens from compound
         _withdrawFromCompoundIfSupported(amountToWithdraw);
+        // TODO Do we need to decrease any amount?
```

Comment:
Please implement this TODO before merging.

Reply:
I created a task to analyze/implement. Now we don't need to get/calculate the total interest withdrawn.